test.sh

```diff
@@ -5,13 +5,13 @@ set -e -x
 export RAILS_ENV=test
 export BUNDLE_WITHOUT=development
 
-npm install
-bundle install
-bundle exec brakeman -z
+npm install -q
+bundle install --quiet
+bundle exec brakeman -q --no-pager
```

Comment:
There was an error on the `--no-pager` flag for the Brakeman gem. Please see the details below.

```
[jonesagyemang:~/Projects/MAS/wpcc] suppress-test-script-output+ 1 ± ./test.sh
+ export RAILS_ENV=test
+ RAILS_ENV=test
+ export BUNDLE_WITHOUT=development
+ BUNDLE_WITHOUT=development
+ npm install -q
+ bundle install --quiet
+ bundle exec brakeman -q --no-pager
invalid option: --no-pager
Please see `brakeman --help` for valid options
```

Reply:
ah, do you want to try updating your brakeman gem @Tr4pSt3R

Reply:
My Brakeman gem version is given below.

```
[jonesagyemang:~/Projects/MAS/wpcc] suppress-test-script-output+ ± bundle exec brakeman -v
brakeman 3.7.0
```

Reply:
 (ruby-2.2.5@wpcc) ~/workspace/wpcc (master)
⤷  brakeman -v
brakeman 3.7.2

Reply:
ah, so mine is now `4.0.1` - I posted in the #backend-developers slack channel about it, but will repost here. There are some changes with the latest version which affect it's use, namely the reason why the `--no-pager` flag has been added. Alternatively we can set the CI environment variable when running the script.

https://brakemanscanner.org/blog/2017/09/25/brakeman-4-dot-0-released/

```Paged Output

By default, output to the terminal will be paged with less or Highline's simple pager.

To disable, use --no-pager.

In 4.0.1 Brakeman will automatically disable the pager when the CI environment variable is set to true. This should be compatible with Travis CI, Circle CI, Codeship, and Bitbucket Pipelines.```

Reply:
@willhall88 since this is so easy to miss can we add this option?

```
# Fail when Brakeman is outdated

bundle exec brakeman -q --no-pager --ensure-latest
```

Reply:
I'd be reluctant to lock the brakeman gem, as the whole point of it is to find security holes and thus is regularly updated with the latest threats

Reply:
ah I didn't know that was a flag we could use. good solution @Tr4pSt3R will test that now